src/cmds/fs/more.c

```diff
@@ -28,29 +28,37 @@ static void screen(FILE *fp) {
 		columns = MAX_SCREEN_WIDTH - 1;
 	}
 	lines = std->endy - std->begy;
-	endwin();
+	//endwin();
 
 	while (1) {
 		for (x = 0; x < lines - 1; x++) {
 
 			for (y = 0; y < columns; y++) {
-				buff[y] = getc(fp);
-				switch ((int) buff[y]) {
-				case EOF:
-					for (i = 0; i < y; i++) {
-						printf("%c", buff[i]);
+				if(fread(&buff[y], 1, 1, fp) == 1) {
```

Comment:
Need to add 1 space here, `if(` -> `if (`. There's no need to hurry up, so take your time to format the code properly.